webapp/src/components/admin_settings/jitsi_section.tsx

```diff
@@ -0,0 +1,271 @@
+                                {'(Optional) The app secret used for authentication by the Jitsi server and JWT token generator.'}
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div className='form-group'>
+                    <label className='col-sm-4'>
+                        {'Meeting Link Expiry Time (minutes):'}
+                    </label>
+                    <div className='col-sm-8'>
+                        <input
+                            className='form-control'
+                            type={'number'}
+                            min={1}
+                            defaultValue={30}
+                            onChange={this.props.onJitsiMeetingLinkExpChange}
+                            value={this.props.meetingLinkExpire}
+                        />
+                        <div className='help-text'>
+                            <span>
+                                {'(Optional) The number of minutes from when the meeting link is created to when it becomes invalid. Minimum is 1 minute. Only applies if using JWT authentication for your Jitsi server.'}
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div className='form-group'>
+                    <label className='col-sm-4'>
+                        {'Enable Compatibility Mode:'}
+                    </label>
+                    <div className='col-sm-8'>
+                        <label className='radio-inline'>
+                            <input
+                                type='radio'
+                                value='true'
+                                checked={this.props.embedded}
+                                onChange={this.props.onJitsiCompatibilityChange}
+                            />
+                            <span>{'true'}</span>
+                        </label>
+                        <label className='radio-inline'>
+                            <input
+                                type='radio'
+                                value='false'
+                                checked={!this.props.embedded}
+                                onChange={this.props.onJitsiCompatibilityChange}
+                            />
+                            <span>{'false'}</span>
+                        </label>
+                        <div className='help-text'>
+                            <span>
+                                {'(Insecure) If your Jitsi server is not compatible with this plugin, include the JavaScript API hosted on your Jitsi server directly in Mattermost instead of the default API version provided by the plugin. WARNING: Enabling this setting can compromise the security of your Mattermost system, if your Jitsi server is not fully trusted and allows direct modification of program files. Use with caution.'}
```

Comment:
I would love to have all this texts translated.

Reply:
> I would love to have all this texts translated.

To add translations for these texts i have to use the Mattermost translation server right? https://translate.mattermost.com/projects/i18n-wip/mattermost-webapp-wip/

Reply:
Not really, we haven't integrated the plugin itself with our weblate, it is just marking the text as translatable (like in other places) and adding the string to the `i18n/en.json` file.